core/utils.js

```diff
@@ -347,23 +347,23 @@ Blockly.createSvgElement = function(name, attrs, parent, opt_workspace) {
 };
 
 /**
- * Deselect any selections on the webpage.
- * Chrome will select text outside the SVG when double-clicking.
- * Deselect this text, so that it doesn't mess up any subsequent drag.
+ * Add a class to the body tag to keep the browser from selecting text, etc.
+ * This is intended to be called when the user initiates some kind of drag event.
+ * 
  */
-Blockly.removeAllRanges = function() {
-  if (window.getSelection) {
-    setTimeout(function() {
-        try {
-          var selection = window.getSelection();
-          if (!selection.isCollapsed) {
-            selection.removeAllRanges();
-          }
-        } catch (e) {
-          // MSIE throws 'error 800a025e' here.
-        }
-      }, 0);
-  }
+Blockly.setPageUnselectable = function() {
+    // don't allow other things to be selected
+    var body = document.getElementsByTagName('body');
+    Blockly.addClass_(body[0], 'blocklyNonSelectable');
+};
+
+/**
+ * Remove the class from the body tag that marks the page unselectable.
+ * This is intended to be called at the end of some kind of drag event.
+ */
+Blockly.setPageSelectable = function() {
```

Comment:
My instinct would be to use `Blockly.setPageSelectable(true)` and `Blockly.setPageSelectable(false)` instead of `Blockly.setPageSelectable()` and `Blockly.setPageUnselectable()`.  But I can't articulate why so I won't argue for it.  What does your gut say?